src/RouteAggregator.js

```diff
@@ -35,40 +37,55 @@ export default class RouteAggregator {
 
       const component = components[i];
```

Comment:
Maybe rename this to `routeComponent` just to avoid the `no-shadow` stuff below?